Doc/library/configparser.rst

```diff
@@ -135,6 +135,29 @@ involves the ``DEFAULT`` section which provides default values for all other
 sections [1]_.  Note also that keys in sections are
 case-insensitive and stored in lowercase [1]_.
 
+If it is necessary to read several configurations, with each one having more
+priority than the previous one, an instance of :class:`ConfigParser` can be
+used to override previously defined properties and retain existing ones.
```

Comment:
One problem I see with this is that the first few clauses of this sentence seem to be introducing a tutorial, i.e. "If you want to do X, you can achieve it with Y", but then it turns out that you are actually *describing the existing behavior* of `ConfigParser` rather than providing a recipe or something.

I would recommend rewriting this as an introduction to the properties of a `ConfigParser`, like so:

> It is also possible to read several configurations into a single ```:class:`ConfigParser` ``` object, in which case the most recent configuration has priority over the previous ones; conflicting keys will be taken from the new configuration and existing keys will be retained.

That's mostly an example, I'm not super happy with that particular wording...

Reply:
@pganssle:
> That's mostly an example, I'm not super happy with that particular wording...

What do you think of this wording?

> It is possible to read several configurations into a single :class:`ConfigParser`, where the most recently added configuration has the highest priority. Any conflicting keys will be taken from the most recent configuration and the previously existing keys will be retained.